packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -251,6 +251,12 @@ namespace Tpetra {
     , haveGlobalConstants_ (false)
     , sortGhostsAssociatedWithEachProcessor_ (true)
   {
+#ifndef TPETRA_ENABLE_DEPRECATED_CODE
+#if (pftype == DynamicProfile)
```

Comment:
See above.  The enum value shouldn't even exist if `TPETRA_ENABLE_DEPRECATED_CODE` is not defined.